modules/programs/inori.nix

```
@@ -0,0 +1,47 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.inori;
+
+  tomlFormat = pkgs.formats.toml { };
+in {
```

style nit: this file is basically `nixpkgs-fmt` compliant, except for here. Personally, I prefer `nixpkgs-fmt` over `nixfmt` and `alejandra`, so I'm happy to keep it this way. If you want to run `nixfmt`, that's also fine.

```suggestion
in
{
```

---

i formatted with ./format from the repo root

---

You're right, sorry about the noise!